tests/setup.py

```diff
@@ -19,7 +19,7 @@
         'xxhash==2.0.2', 'protobuf==4.21.8', 'fastavro==1.4.9',
         'psutil==5.9.0', 'numpy==1.22.3', 'pygal==3.0', 'pytest==7.1.2',
         'jump-consistent-hash==3.2.0', 'azure-storage-blob==12.14.1',
-        'kafkatest@git+https://github.com/apache/kafka.git@058589b03db686803b33052d574ce887fb5cfbd1#egg=kafkatest&subdirectory=tests',
+        'kafkatest@git+https://github.com/apache/kafka.git@d8d92f0f8014e5686e956a0c82bb2afdd41592d5#egg=kafkatest&subdirectory=tests',
```

Comment:
I didn't see a specific change related to ducktape dependency in this commit, nor could I find an associated tag, would you help me identify which version this is?

Reply:
I'll change it to  https://github.com/apache/kafka/commit/7e683852b46c0fe70f5100e953e3cd61d1697c06

Reply:
Would it not make sense to bump it to some version that has a release tag? Perhaps 3.2.0 (first release tag containing the fix), 3.2.3 (latest patch on that branch), or 3.5.1 (latest tag)?

Reply:
done